lib/mayaUsd/ufe/UsdSceneItem.cpp

```diff
@@ -62,8 +62,7 @@ std::vector<std::string> UsdSceneItem::ancestorNodeTypes() const
 	const TfType& schemaType = fPrim.GetPrimTypeInfo().GetSchemaType();
 #endif
 	if (!schemaType) {
-		TF_CODING_ERROR("Could not find prim type '%s' for prim %s",
-			fPrim.GetTypeName().GetText(), UsdDescribe(fPrim).c_str());
```

Comment:
This error was too extreme. There are nodes which don't have a schema type, so just silently return empty ancestor types.